build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# ****************************************************************************
+# Copyright 2004-2013 Castle Project - http://www.castleproject.org/
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ****************************************************************************
+
+clear
+
+DOTNETPATH=$(which dotnet)
+
+if [ ! -f "$DOTNETPATH" ]; then
+
+	echo "Please install Microsoft/netcore from: https://www.microsoft.com/net/core"
+
+	exit 1
+
+fi
+
+MONOPATH=$(which mono)
+
+if [ ! -f "$MONOPATH" ]; then
+
+	echo "Please install Xamarin/mono from: http://www.mono-project.com/docs/getting-started/install/"
+
+	exit 1
+
+fi
+
+dotnet restore ./buildscripts/BuildScripts.csproj -s https://dotnet.myget.org/F/dotnet-core/api/v3/index.json
```

Comment:
I'm guessing the `Microsoft.TargetingPack.NETFramework.v4.6.1` package comes from this NuGet repo. What does the package do and why is it only needed on Linux?